backend/app/llm/__init__.py

```
@@ -1,9 +1,9 @@
 """
 Sanctum LLM Provider Module
 
-Generic OpenAI-compatible interface supporting multiple providers.
+Maple-first OpenAI-compatible LLM interface.
 """
 
-from .provider import LLMProvider, LLMResponse, get_provider
+from .provider import LLMProvider, LLMResponse, get_provider, get_maple_provider
 
-__all__ = ["LLMProvider", "LLMResponse", "get_provider"]
+__all__ = ["LLMProvider", "LLMResponse", "get_provider", "get_maple_provider"]
```

Ruff (0.15.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)